### ISSUE TITLE
Error: [ethjs-rpc] rpc error with payload when used on Kovan or Mainnet

### ISSUE THREAD

**Issue by mattvick** [ethjs/ethjs-signer#2]
# ethjs-signer

## Issue Type

- [x] Bug (https://github.com/ethjs/ethjs-signer/blob/master/.github/CONTRIBUTING.md#bug-reports)
- [ ] Feature (https://github.com/ethjs/ethjs-signer/blob/master/.github/CONTRIBUTING.md#feature-requests)

## Description

An `Error: [ethjs-rpc] rpc error with payload` error is thrown when implementing the code from the usage instructions section of the README on both the **Kovan** and **Main** networks.

```
Error: [ethjs-rpc] rpc error with payload {"id":339528...,"jsonrpc":"2.0","params":["0xf86..."],"method":"eth_sendRawTransaction"} null
    at bundle.js:13279
    at XMLHttpRequest.request.onreadystatechange (bundle.js:8474)
```

This appears to be a `ethjs-rpc` error, but it is happening when signing transactions using the `ethjs-signer` package.

## Steps to reproduce

- Copy the usage code example from the README
- Update the HttpProvider replacing the localhost URL `new HttpProvider('http://localhost:8545')` with the Kovan testnet or the mainnet URL for e.g. `new HttpProvider('https://kovan.infura.io/AccessTokenHere')`
- Replace the account address and private key with details of a real account with an Eth balance
- Webpack and run the code

# Versions

- Node/NPM: Node v6.9.1 / NPM 3.10.8
- Browser: Chrome version 57.0.2987.133 (64-bit)
- OS: macOS Sierra version 10.12.4 (16E195)

**Comment by SilentCicero** [comment]
@mattvick will review, the code is simple enough if you find a fix just make a PR.

**Comment by mattvick** [comment]
There were a couple of things I did that caused this:

1. I was not defining the `gasPrice` when creating my transactions, which meant they were not executed and became 'stuck' 
2. When creating new transactions I was incrementing the nonce manually, meant that several 'stuck' transactions were created that needed to be overwritten

To assist others I've added a the missing `gasPrice`definition to the usage example in the README.md and a note that this is required.

Credit goes @arkpar who pointed me in the right direction on Gitter
